### ISSUE TITLE
abstract methods are not in definition files

### ISSUE THREAD

**Issue by Dinistro** [angular/angular#10040]
**I'm submitting a ...**  
```
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
The definition files are missing the abstract methods. This can result in compile exceptions, when I'm doing stuff like an `updateValue()` on an NgControl.control.

**Reproduction of the problem**

Here you can see code that works perfectly fine, but does not compile: 

```
//this.control is type of NgControl
this.control.control.updateValue(valueStr);
```

https://github.com/valor-software/ng2-bootstrap/pull/723/files#diff-41c9e3a2e03ed38fe48c8a2a9dfa600aR176

The message says: [Property 'updateValue' does not exist on type 'AbstractControl'](https://travis-ci.org/valor-software/ng2-bootstrap/builds/144189770#L809)

But if you take a look at the source, there it is: https://github.com/angular/angular/blob/master/modules/%40angular/forms/src/model.ts#L176

**What is the expected behavior?**
the abstract methods should also be part of the `.d.ts` files.

**What is the motivation / use case for changing the behavior?**

This would make it much easier to work with the abstract angular classes. For example reusable form components like the example. 

**Please tell us about your environment:**

* **Angular version:** 2.0.0-rc.4
 
* **Language:** TypeScript 1.8.10

**Comment by rkirov** [comment]
That's a bug, TypeScript (at least 1.9+) keeps abstract properties in the definitions, when emitting .d.ts files with -d. No idea who is doing the stripping. Conceptually, stripping abstract properties is wrong, because it produces the wrong type shape for the abstract base class.

Maybe @alexeagle knows?

**Comment by jasonchoimtt** [comment]
I think it's just that the change in https://github.com/angular/angular/pull/9901 has not landed in npm yet. Can @kara confirm?

**Comment by alexeagle** [comment]
You can see the `.d.ts` files in our snapshots do have the abstract methods:
https://github.com/angular/forms-builds/blob/master/src/model.d.ts

(we strip the `abstract` keywords from the `.d.ts` files before publishing to npm, because it's not available in TypeScript 1.8 which is still the GA version we expect users to depend on.
https://github.com/angular/angular/blob/6bfd514caf56f49dc51c650f64fdb2c5ab9b6019/build.sh#L78
but we only strip the `abstract` keyword, not the whole method declaration.)

So I agree with @jasonchoimtt that it just looks like version skew, the .d.ts files published in rc4 don't reflect the API change
